packages/expo-env-info/jest.config.js

```diff
@@ -1,7 +1,11 @@
-const path = require('path');
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 
-module.exports = {
+export default {
```

Comment:
Isn't it a problem that this is in a `.js` file? I mean it'd be probably better to rename it if it's possible.
I remember I hit this when I was trying to convert EAS CLI to ESM. But I don't remember the details.

Reply:
The config is being read 'in ESM mode' as well, not changing this will throw an error about it.